agent/src/beerocks/slave/agent_ucc_listener.cpp

```diff
@@ -192,6 +192,13 @@ bool agent_ucc_listener::handle_dev_set_config(std::unordered_map<std::string, s
 
     auto &backhaul_param = params["backhaul"];
     std::transform(backhaul_param.begin(), backhaul_param.end(), backhaul_param.begin(), ::tolower);
+    // TODO - add wireless backhaul support.
+    // For now, this causes slave reset and the test to hang,
+    // so return not supported.
+    if (backhaul_param != "eth") {
```

Comment:
LGTM, but wondering if `backhaul_param` can also be empty..

Reply:
It cannot, its invalid if it is